src/test-suite/react-todoapp.tsx

```diff
@@ -53,7 +54,8 @@ export class TodoApp extends React.Component<TodoAppProps, any> {
 		<div>
 				<header>
 					<input value={this.state.newItem} onChange={(e) => this.setState({newItem: e.target.value})}/>
-					<button className='add' onClick={this.onAdd}>Add</button>
+					<button className='add' onClick={this.onAdd} onMouseEnter={(e) => this.setState({event: e.type})}>Add</button>
+					{this.state.event && <label className='event'>{this.state.event}</label>}
```

Comment:
True. I asked @itaibenda the exact same question and our assumption was that the whole purpose of the todo-app is to allow testing.

Creating another app might cause confusion regarding where new features should be tested.

Again, I am open to all options. Feel free to suggest and I will try to act accordingly.

Reply:
that was its purpose indeed
my point is that you wouldn't find a "show event type on a label" feature in a todo app, thus I would think of:
1) adding a real "hovering" feature to the todo app (shows details on hover? shows actions menu on hover?)
2) adding another, more technical test app in parallel

I think that while option #2 might be needed in the feature for more indicative failures, the right place to start with is #1 which better matches the values of this library - real features / real users

also this might indicate this test:
```
assert.equal(await driver.$('.event').text(), 'mouseenter');
```
might indicate we're testing implementation instead of the feature (if chrome decides to fire mouseover after mouseenter pup. tests will fail on this)